### PR TITLE
Add transaction

### PR DESCRIPTION
**Pull request by novikovalexandr**

### CODE REVIEW

File: app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsViewModelImpl.java [hunk 1]
```diff
@@ -108,9 +140,34 @@ private Wallet randomWallet(CoinEntity coin) {
         return new Wallet(UUID.randomUUID().toString(), coin.id, 10 * random.nextDouble());
     }
 
+    private List<Transaction> randomTransactions(Wallet wallet) {
+        List<Transaction> transactions = new ArrayList<>();
+        int countTransaction = 20;
```
Comment (askden8) [inline app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsViewModelImpl.java:145]: could it be static variable?

File: app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsViewModelImpl.java [hunk 2]
```diff
@@ -108,9 +140,34 @@ private Wallet randomWallet(CoinEntity coin) {
...
+        }
+
+        return transactions;
+    }
+
+    private Transaction randomTransaction(Wallet wallet) {
+        Random random = new Random();
+
+        int maxCountCoin = 4;
+        long startDate = 1546300800000L;
```
Comment (askden8) [inline app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsViewModelImpl.java:158]: pls not use "magic numbers"

File: app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsViewModelImpl.java (hunk 1 above)
Comment (novikovalexandr) [reply app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsViewModelImpl.java:145]: changed

File: app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsViewModelImpl.java (hunk 2 above)
Comment (novikovalexandr) [reply app/src/main/java/com/alexandr4/loftcoin/screens/wallets/WalletsViewModelImpl.java:158]: changed

**Pull request merged**